dev-tools/Jenkinsfile.yml

```diff
@@ -0,0 +1,15 @@
+when:
+    branches: true             ## for all the branches
+    changeset:                 ## when PR contains any of those entries in the changeset
+        - "^dev-tools/.*"
```

Comment:
Some things here also depend on libbeat, including the `make check` itself.

Reply:
so, Shall we add `^Makefile` and `^libbeat/.*`?

Reply:
I don't think `^Makefile` is needed, I meant that it depends on the Makefile in libbeat, in `libbeat/scripts/Makefile`.